File: problem-1/prb1.py
# word is the key
keyword = {"break", "case", "continue", "default", "defer", "else", "for", "func", "goto", "if", "map", "range", "return", "struct", "type", "var"}
u_word = input()
if u_word in keyword:
    print(u_word + " is a keyword")
else:
    print(u_word + " is not a keyword")

File: problem-1/prb1.c
#include<stdio.h>
#include<string.h>
int main()
{
    char str[16][10]={"break", "case", "continue", "default", "defer", "else", "for", "func", "goto", "if", "map", "range", "return", "struct", "type", "var"};
    char input[20];
    scanf("%s",input);
    int flag=0;
    for(int i=0;i<16;i++)
    {
        if(strcmp(input,str[i])==0)
        {
            flag=1;
            break;
        }
    }
    if(flag==1)
    {
        printf("%s is a keyword",input);
    }
    else
    {
        printf("%s is not a keyword",input);
    }
    return 0;
}
